### PR TITLE
Clarify the scope of these standards in the top-level readme.

### PR DESCRIPTION
**Pull request by samcunliffe**
Sam's attempt at a clear and charming summary of the reasoning for publishing a style guide. Picky comments about language are very welcome. I wanted to be a bit more brief, but I couldn't think of a way.

### Note

In the meeting we actually said we "should" use them unless there was a good reason. But "should not" change for existing projects.

### Also

Emoji's in a public-facing site: yay or nay? It's very on-brand for ARC. So I went with yay.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,36 @@
...
+
+They represent our default style and standard choices that our projects should
+use if there are no other constraints. If you are working with us, we'll
+probably start using these standards and may point you to this page (👋).
+
+In the best case, this means that there is only one standard to learn per language.
+We hope this results in:
+  * 😊 less time thinking about style,
+  * 💪 faster onboarding of staff to new projects, and
+  * 🚀 faster review of code.
```
Comment (stefpiatek) [inline README.md:13]: maybe consistent configuration of tooling to ensure standards are being met at as well?

**Review by stefpiatek — APPROVED** [review]
Chef's kiss, thanks Sam

**Review by milanmlft — APPROVED** [review]
Molto bene 👌

File: README.md (hunk 1 above)
Comment (samcunliffe) [reply README.md:13]: Is just
```
* 🛠️ a more consistent tooling setup,
```
clear enough?

**Review by p-j-smith — APPROVED** [review]
lovely stuff

File: README.md (hunk 1 above)
Comment (stefpiatek) [reply README.md:13]: Yeah like it

**Review by matt-graham — APPROVED** [review]
Looks great. Also very much pro emojis 😁

**Pull request merged**